director/origin_api_test.go

```diff
@@ -69,48 +69,38 @@ func TestVerifyAdvertiseToken(t *testing.T) {
 	assert.NoError(t, err)
 	assert.Equal(t, true, ok, "Expected scope to be 'pelican.advertise'")
 
-	//Create token without a scope - should return an error
-	key, err := config.GetIssuerPrivateJWK()
-	err = jwk.AssignKeyID(key)
-	assert.NoError(t, err)
-
-	scopelessTok, err := jwt.NewBuilder().
-		Issuer("").
-		Audience([]string{"director.test"}).
-		Subject("origin").
-		Build()
+	//Create token without a scope - should return an error upon validation
+	scopelessTokCfg := utils.TokenConfig{
+		TokenProfile: utils.WLCG,
```

Comment:
For cases where we haven't used `scitoken` or `WLCG` as the token profile, I'd suggest using `utils.None` as the token profile now unless we want to take this chance to enforce all the token in Pelican to adhere to a certain token profile, which I feel should be documented somewhere to let people know

Reply:
I do like the idea of conforming to some kind of well-established and well-studied token scheme, even for internal stuff. Where would you recommend documenting this change? The dual of this is that wherever we add a token profile where there wasn't one before, the receiver of the token should also double check that the token conforms to the profile. Let's open that as an issue after this is merged.

Reply:
I would say in `pelican-dev` channel? Agree that we should enforce the check at the issuer end as well.